floodlight/models/utils.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+
+from floodlight import XY
+
+
+def _exclude_x_ids(xy: XY, xIDs_to_exclude: list) -> np.ndarray:
+    """Returns an array of boolean values indicating which xIDs to exclude.
+
+    Parameters
+    ----------
+    xy: XY
+        Player spatiotemporal data from which xIDs should be excluded.
```

Comment:
seems a bit overkill for this function to pass an xy object, as essentially just the ```N```-property is needed to juggle some indices, but I guess it's fine